pages/account/manage/event/[[...data]].js

```diff
@@ -242,7 +243,7 @@ export default function ManageEvent({ BASE_URL, event }) {
                       type="number"
                       name="price"
                       label="Ticket Price"
-                      onChange={(e) => setPrice(e.target.value)}
+                      onChange={(e) => setPrice(e.target.value < 0 ? 0 : e.target.value)}
```

Comment:
I think the better solution would be to add validation rather than change the number.

This can be done with `min` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

Reply:
Thank you for this suggestion. I've made the change accordingly 👍 
![image](https://github.com/EddieHubCommunity/LinkFree/assets/121224710/e85411a6-a2c7-46e4-8b35-6b92c41dc9f4)